### PR TITLE
Update CODEOWNERS file

### PR DESCRIPTION
**Pull request by nick-mobilecoin**

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #14** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/14" target="_blank">Graphite</a> 
    * **PR #13** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/13" target="_blank">Graphite</a> 
      * **PR #15** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/15" target="_blank">Graphite</a> 
        * **PR #22** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/22" target="_blank">Graphite</a> 
          * **PR #24** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/24" target="_blank">Graphite</a> 
            * **PR #30** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/30" target="_blank">Graphite</a> 
              * **PR #31** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/31" target="_blank">Graphite</a> 
                * **PR #32** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/32" target="_blank">Graphite</a>  👈
        * **PR #21** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/21" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/32?utm_source=stack-comment).

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,7 @@
-* @mobilecoinfoundation/coredev
+/alloc/         @nick-mobilecoin
+/.github/       @jcape @nick-mobilecoin
+/io/            @nick-mobilecoin
+/panic/         @nick-mobilecoin
+/test_enclave/  @nick-mobilecoin
+/LICENSE        @jcape
+/*.* @mobilecoinfoundation/coredev
```
Comment (nick-mobilecoin) [inline .github/CODEOWNERS:7]: @jcape the owners file in SGX had you listed as the default do we want that here too or should it be the team in SGX?

Comment (jcape) [reply .github/CODEOWNERS:7]: I'm fine to take it here as well, yeah.

Comment (nick-mobilecoin) [reply .github/CODEOWNERS]: updated to use jcap for fallback

**Review by samdealy — APPROVED** [review]
LGTM

**Review by awygle — APPROVED** [review]

**Comment by nick-mobilecoin** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: 174FRuRmqv3ul7Wrh6cz.
See details on [graphite.dev](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx-std/32)

**Comment by nick-mobilecoin** [comment]
@awygle bumping for re-review since I did a restack

**Review by awygle — APPROVED** [review]

**Pull request merged**